app/validate/hour_validator.py

```
@@ -52,17 +55,15 @@ def validate_hour_continuous(hour_slots: List[str], date: str):
-    times.sort()
-
-    # 인접한 시간 간격이 1시간(1시간 간격의 연속)인지 확인
-    for i in range(len(times) - 1):
-        dt_current = datetime.combine(datetime.today(), times[i])
-        dt_next = datetime.combine(datetime.today(), times[i + 1])
-        diff = dt_next - dt_current
-        if diff != timedelta(hours=1):
+    # 인접한 시간 간격이 1시간(60분)인지 확인
+    for i in range(len(slots) - 1):
+        if slots[i + 1] - slots[i] != 60:
             raise HourDiscontinuousError(f"시간 슬롯이 1시간 단위로 연속적이지 않습니다.")
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)